src/renderer/Components/CreatingPackage/PackagePreview/Order/index.jsx

```diff
@@ -19,11 +26,22 @@ function Order({ order, index }) {
         </div>
       </div>
       <div className="mx-auto my-2 flex w-fit flex-row items-center justify-start justify-items-start space-x-4 rounded border border-solid border-gray-300 p-2">
-        {sourcePath && <div className="text-block-blue">{sourcePath} 의</div>}
-        <div className="text-block-blue">{attachmentName} 을(를)</div>
-        <div className="text-block-blue">{executionPath} 에</div>
-        {editingName && (
-          <div className="text-block-blue">{editingName} 으로</div>
+        {sourcePath && attachmentType !== "folder" && (
+          <div className="text-block-blue">{sourcePath} 의</div>
+        )}
+        <div className="text-block-blue">
+          {attachmentType === "folder"
+            ? sourcePath || executionPath
+            : attachmentName}{" "}
```

Comment:
```suggestion
            : attachmentName}
```
`{"  "}` 는 꼭 필요한 부분 인가요? 필요하다면 합쳐서 한꺼번에 뒤의 문자열과 나열할 수 있을 것 같습니다. 😊

Reply:
오 확인하고 아래와 같이 수정했습니다.👍
![image](https://github.com/user-attachments/assets/786f3f6f-2ae3-49f7-8c1a-b10c6b0dee70)

Reply:
좋습니다 :)